sagemaker-pyspark-sdk/tox.ini

```diff
@@ -1,5 +1,10 @@
 [tox]
-envlist = clean,flake8,docs,py27,py35,py36,stats
+envlist = clean,flake8,docs,py{27,35,36},stats
```

Comment:
Do we have to keep these versions in sync with the .travis.yml file? We have py27, 35 and 36 here and 27, 34 and 35 in the yml file.

Reply:
Good catch. Updating travis python version to align with tox versions.